src/node_worker.cc

```diff
@@ -163,10 +180,73 @@ void Worker::Run() {
     SealHandleScope outer_seal(isolate_);
     bool inspector_started = false;
 
+    DeleteFnPtr<Environment, FreeEnvironment> env_;
+    OnScopeLeave cleanup_env([&]{
```

Comment:
`[&]() {`? That's seems to be the prevailing way of writing that in the source tree.